src/modules/core/components/emoji/emoji.interface.ts

```diff
@@ -0,0 +1,5 @@
+export type EmojiProps = {
+  name: 'sunglasses' | 'heart_eyes';
```

Comment:
Create a separate type `EmojiName`:

```
export type EmojiName = 'sunglasses' | 'heart-eyes';

export type EmojiProps = {
  name: EmojiName;
  ...
}
```

And sorry, it was my mistake but rename `heart_eyes` to `heart-eyes`.

Reply:
I suggest that we use same approach that we used in the `Icon` component instead, in that approach EmojiName will generate automatically.